packages/manager/src/features/Linodes/LinodesDetail/LinodeAlerts/AlertsPanel.tsx

```diff
@@ -319,6 +337,24 @@ export const AlertsPanel = (props: Props) => {
         )}
       </Prompt>
 
+      {/* Save Legacy Alerts Confirmation Modal. This modal appears on "Save" only
+      when user already subscribed to Beta/ACLP Mode and makes changes in the
+      Legacy mode Interface. */}
+      <AlertConfirmationDialog
+        handleCancel={() => setIsDialogOpen(false)}
+        handleConfirm={() => formik.handleSubmit()}
+        isLoading={isPending}
+        isOpen={isDialogOpen && isLinodeAclpSubscribed}
+        message={
+          <>
+            Are you sure you want to save Legacy Alerts? <b>Alerts(Beta)</b>{' '}
+            settings will be disabled and replaced by the original Legacy
+            settings.
+          </>
+        }
+        primaryButtonLabel="Confirm"
+        title="Are you sure you want to save Legacy Alerts?"
```

Comment:
```suggestion
            Are you sure you want to save legacy Alerts? <b>Alerts(Beta)</b>{' '}
            settings will be disabled and replaced by the original legacy
            settings.
          </>
        }
        primaryButtonLabel="Confirm"
        title="Are you sure you want to save legacy Alerts?"
```

Should "legacy" not be capitalized here for consistency with the rest of the feature? 

Not capitalized:
![Screenshot 2025-07-15 at 10 06 06 AM](https://github.com/user-attachments/assets/06591386-b66e-45e0-8faa-e0b8a9fca660)
![Screenshot 2025-07-15 at 10 06 17 AM](https://github.com/user-attachments/assets/8f8a36d1-ef72-4946-a443-36699975d290)

Reply:
my https://github.com/linode/manager/pull/12511 pr refactors the strings into a constants file, but i didn't include this one bc i didn't know how given the tags within the string. if there's an elegant way to do this pls let me knwo.

Reply:
One thing that comes to mind is renaming the file from `constants.ts` to `constants.tsx` and scoping React in that file. That way, you can use tags if needed, and we can define constants that include React nodes. For eg.:
```
export const ALERTS_LEGACY_CONFIRMATION_MODAL_TEXT: ReactNode = (
  <>
    Are you sure you want to save Legacy Alerts? <b>Alerts(Beta)</b> settings
    will be disabled and replaced by the original Legacy settings.
  </>
);
```

Reply:
Note: If these constants are only used for assertions in tests and not rendered directly in components, there's no need to define them as ReactNode. They can be stored as plain strings instead.

Reply:
Got confirmation! I've updated the text as per the latest UX mocks